tests/test_ops.py

```diff
@@ -0,0 +1,121 @@
+"""
+Unit tests for GPI2 operation and its utility functions.
+"""
+import pytest
+import torch
+import jax
+
+import pennylane as qml
+from pennylane import numpy as np
+import pennylane.math as math
+import tensorflow as tf
+from ionizer.ops import GPI
+
+jax.config.update("jax_enable_x64", True)
+
+interfaces = [
```

Comment:
To avoid typing everything out again (and since you two are sitting right next to each other :laughing: ) please see the comments I've left on [the GPI2 PR](https://github.com/QSAR-UBC/ionizer/pull/2/files) regarding how to structure the interface testing, as well as trying with multiple diff methods.